packages/stark-ui/src/modules/restrict-input-directive.ts

```diff
@@ -0,0 +1 @@
+export * from "./restrict-input-directive/restrict-input-directive.module";
```

Comment:
We should also re-export the `directives` sub-folder here:
`export * from "./restrict-input-directive/directives";`

I guess that is what making Travis fail too

Reply:
👍 I updated the PR